src/axolotl/monkeypatch/fastchat_conversation_turns.py

```diff
@@ -88,7 +88,7 @@ def get_turns(  # pylint: disable=too-many-return-statements
             yield "", system_prompt
         else:
             yield "", "[INST] "
-        for i, (role, message) in enumerate(self.messages[1:]):
+        for i, (role, message) in enumerate(self.messages):
             if message:
                 yield role + " ", message + seps[i % 2]
```

Comment:
Thanks, I noticed the first message was removed as well.

I also think this needs to change as well be consistent with: https://github.com/lm-sys/FastChat/blob/main/fastchat/conversation.py#L135

```suggestion
                if i == 0:
                    yield "", message + " "
                else:
                    yield role + " ", message + seps[i % 2]
```

then it should match the pseudocode in: https://huggingface.co/mistralai/Mixtral-8x7B-Instruct-v0.1

Reply:
This actually adds more artifacts to the prompt that I don't like.  Adds this to the end of each sequence (overkill):

`</s><s></s>`

Reply:
Hmmm actually it might have already been there before this PR and just noticed this now!  ok  nevermind this is something else that I need to fix!